.github/workflows/deploy-wiab.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy on Hetzner WIAB setup
+on:
+  workflow_run:
+    workflows: ["Prepare custom offline package"]
+    types:
+      - completed
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: autodeploy-script
+      cancel-in-progress: false
+
+    steps:
+      # Step 1: Checkout the repository code
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # Step 2: Set up SSH key for remote access
+      - name: Set up SSH key
+        uses: webfactory/ssh-agent@v0.5.3
+        with:
+          ssh-private-key: ${{ secrets.WIAB_PRIVATE_SSH_KEY }}
+    
+      # Step 3: Get the latest commit SHA, for the artifact
+      - name: Get latest commit SHA
+        id: get_commit_sha
+        run: |
+          COMMIT_SHA=$(git rev-parse HEAD)
+          echo "commit_sha=$COMMIT_SHA" >> $GITHUB_ENV
+
+      # Step 4: Run the autodeploy script
+      - name: Run Auto Deploy Script
+        run: |
+          cd bin
+          ./autodeploy.sh --artifact-hash 2e8e4319ff3a8c309dc0ef5a1800611faa1ab0cb --target-domain wiab-test-box.wire.link --force-redeploy
```

Comment:
Shouldn't the `artifact-hash` ID be invoked dynamically from `$COMMIT_SHA` above?

Reply:
nice catch, thanks. I forgot to remove the hard coded sha used for testing. Fixed now